### DISCUSSION TITLE
`reactpy-material`: Using Material UI components

### DISCUSSION THREAD

**Discussion by williamneto** [idom-team/idom#1181] [Downloadable Packages]
Hi everyone,

I just started a project to extend the capabilities of reactpy in the design field, it is [reactpy-material](https://github.com/williamneto/reactpy-material). The idea is bring Material UI components to reactpy projects.

Material its a popular UI and very usefull for when you want to start something and doesnt care too much about the design. With its ready to use UI components its easyer to build interfaces.

By now there are only 3 components but i intend to keep bringing more.

Any review or sugestion is welcome

[reactpy-material](https://github.com/williamneto/reactpy-material)

**Comment by Archmonger** [comment]
Thank you so much for starting this project!

There's [a lot of people](https://github.com/reactive-python/reactpy/discussions?discussions_q=is%3Aopen+material+) who have asked about MaterialUI. I highly recommend commenting on their posts to advertise the project.
